modules/core/src/main/java/org/apache/ignite/cache/query/SqlFieldsQuery.java

```diff
@@ -204,6 +208,42 @@ public SqlFieldsQuery setTimeout(int timeout, TimeUnit timeUnit) {
         return this;
     }
 
+    /**
+     * Sets server side update flag.
```

Comment:
Let's be more specific. 
E.g. "Sets "skipReducerOnUpdate" query flag." 
or "Switches update mode on server side."